### PR TITLE
AT_01.03_021| Verify the "Credentials" in dropdown menu is clickable

### PR DESCRIPTION
**Pull request by AnastasiiaNY**

### CODE REVIEW

**Review by Rom-T — APPROVED** [review]

**Comment by S-Lanchik** [comment]
Put your AT link

File: cypress/e2e/userIconHeader.cy.js [hunk 1]
```diff
@@ -20,5 +25,13 @@
         cy.get('.yuimenuitemlabel span').contains('Credentials').click()
         cy.url().should('includes','/credentials/')  
     })
+    
+    it('AT_01.03_021| Verify the "Credentials" in dropdown menu is clickable',()=>{
+        cy.get (`a[href="/user/${USERNAME}`).should('be.visible')
+        cy.get ('.login .jenkins-menu-dropdown-chevron').click({force:true})
+        cy.get ('.login .jenkins-menu-dropdown-chevron').should('be.visible')
+        cy.contains('Credentials').click({force:true})
```
Comment (OLgaByR) [inline cypress/e2e/userIconHeader.cy.js:33]: 'Credentials' text should be placed in fixtures

File: cypress/e2e/userIconHeader.cy.js [hunk 2]
```diff
@@ -20,5 +25,13 @@
         cy.get('.yuimenuitemlabel span').contains('Credentials').click()
         cy.url().should('includes','/credentials/')  
     })
+    
+    it('AT_01.03_021| Verify the "Credentials" in dropdown menu is clickable',()=>{
+        cy.get (`a[href="/user/${USERNAME}`).should('be.visible')
+        cy.get ('.login .jenkins-menu-dropdown-chevron').click({force:true})
```
Comment (SvetlanaVet) [inline cypress/e2e/userIconHeader.cy.js:31]: It's not good practice to use {force:true}, please will find the correct selector next time

**Comment by AnastasiiaNY** [comment]
[AT_01.03_021| <Manage my user icon>](https://trello.com/c/qgMI5OlL/765-at0103021-manage-my-user-icon)

**Pull request merged**